components/image-control/style.scss

```diff
@@ -0,0 +1,4 @@
+.blocks-textarea-control__input {
+	width: 100%;
+	padding: 6px 8px;
+}
```

Comment:
This appears to be styling which applies to the `TextareaControl` rather than this component.

Reply:
Oops, Will fix 🛠

Reply:
Done ;) Removed the style.scss file...

Reply:
@paulwilde Fixed the error in code...